client/views/windows/settings.tpl

```diff
@@ -187,17 +187,20 @@
 				<div class="col-sm-12">
 					<h2>Change password</h2>
 				</div>
-				<div class="col-sm-12">
+				<div class="col-sm-12 password-container">
 					<label for="old_password_input" class="sr-only">Enter current password</label>
 					<input type="password" id="old_password_input" name="old_password" class="input" placeholder="Enter current password">
+					<i class="see-pw" title="Show Password"></i>
 				</div>
-				<div class="col-sm-12">
+				<div class="col-sm-12 password-container">
 					<label for="new_password_input" class="sr-only">Enter desired new password</label>
 					<input type="password" id="new_password_input" name="new_password" class="input" placeholder="Enter desired new password">
+					<i class="see-pw" title="Show Password"></i>
 				</div>
-				<div class="col-sm-12">
+				<div class="col-sm-12 password-container">
 					<label for="verify_password_input" class="sr-only">Repeat new password</label>
 					<input type="password" id="verify_password_input" name="verify_password" class="input" placeholder="Repeat new password">
+					<i class="see-pw" title="Show Password"></i>
```

Comment:
- Use `aria-label` (you can also add tooltipped class to show a proper tooltip).
- Put `Hide password in an alt-attribute something like that: https://github.com/thelounge/thelounge/blob/49c5e6a502bf3261302386143b89d49262a23fe0/client/js/socket-events/more.js#L118
- Since `see-pw` element is a little elaborate, you could make a separate template file for it.
- Change the classname to something more verbose like `reveal-password`.